class.jetpack.php

```diff
@@ -4032,6 +4032,24 @@ function admin_page_load() {
 					wp_safe_redirect( Jetpack::admin_url( array( 'page' => $redirect ) ) );
 				}
 				exit;
+			case 'onboard' :
+				$user_can = is_multisite()
+					? current_user_can_for_blog( get_current_blog_id(), 'manage_options' )
+					: current_user_can( 'manage_options' );
```

Comment:
My WP-fu is rusty. Is there no shorthand for constructs like this? The single-site/multi-site `current_user_can()` check I mean...

Reply:
No need to have a check for multisite, simplified in https://github.com/Automattic/jetpack/pull/8352/commits/ae2284fb9d1f6ddf003e805409be7dd615d8f4a2.